federation/pkg/federation-controller/secret/secret_controller.go

```diff
@@ -241,189 +251,184 @@ func (secretcontroller *SecretController) removeFinalizerFunc(obj pkgruntime.Obj
 		return obj, nil
 	}
 	meta.Finalizers = newFinalizers
-	secret, err := secretcontroller.adapter.FedUpdate(obj)
+	secret, err := s.adapter.FedUpdate(obj)
 	if err != nil {
-		return nil, fmt.Errorf("failed to remove finalizer %s from %s %s: %v", finalizer, adapter.Kind(), meta.Name, err)
+		return nil, fmt.Errorf("failed to remove finalizer %s from %s %s: %v", finalizer, s.adapter.Kind(), meta.Name, err)
 	}
 	return secret, nil
 }
 
 // Adds the given finalizers to the given objects ObjectMeta.
-// Assumes that the given object is a secret.
-func (secretcontroller *SecretController) addFinalizerFunc(obj pkgruntime.Object, finalizers []string) (pkgruntime.Object, error) {
-	adapter := secretcontroller.adapter
-	meta := adapter.ObjectMeta(obj)
+func (s *FederationSyncController) addFinalizerFunc(obj pkgruntime.Object, finalizers []string) (pkgruntime.Object, error) {
+	meta := s.adapter.ObjectMeta(obj)
 	meta.Finalizers = append(meta.Finalizers, finalizers...)
-	secret, err := secretcontroller.adapter.FedUpdate(obj)
+	secret, err := s.adapter.FedUpdate(obj)
 	if err != nil {
-		return nil, fmt.Errorf("failed to add finalizers %v to %s %s: %v", finalizers, adapter.Kind(), meta.Name, err)
+		return nil, fmt.Errorf("failed to add finalizers %v to %s %s: %v", finalizers, s.adapter.Kind(), meta.Name, err)
 	}
 	return secret, nil
 }
 
-func (secretcontroller *SecretController) Run(stopChan <-chan struct{}) {
-	go secretcontroller.secretInformerController.Run(stopChan)
-	secretcontroller.secretFederatedInformer.Start()
+func (s *FederationSyncController) Run(stopChan <-chan struct{}) {
+	go s.controller.Run(stopChan)
+	s.informer.Start()
 	go func() {
 		<-stopChan
-		secretcontroller.secretFederatedInformer.Stop()
+		s.informer.Stop()
 	}()
-	secretcontroller.secretDeliverer.StartWithHandler(func(item *util.DelayingDelivererItem) {
-		secret := item.Value.(*types.NamespacedName)
-		secretcontroller.reconcileSecret(*secret)
+	s.deliverer.StartWithHandler(func(item *util.DelayingDelivererItem) {
+		nsName := item.Value.(*types.NamespacedName)
```

Comment:
(Here and everywhere, I think: the difference between "namespace name" and "namespaced name" is significant enough that using nsName for both seems like it decreases readability.